backend/apps/monkvision/lib/app.js

```diff
@@ -2,4 +2,8 @@
  * (C) 2020 TekMonks. All rights reserved.
  */
 
-module.exports.initSync = _ => { global.APP_CONSTANTS = require(`${__dirname}/../apis/lib/constants.js`); require(`${APP_CONSTANTS.LIB_DIR}/db.js`).init(); }
\ No newline at end of file
+module.exports.initSync = _ => {
+    global.APP_CONSTANTS = require(`${__dirname}/../apis/lib/constants.js`);
+    require(`${APP_CONSTANTS.LIB_DIR}/db.js`).init();
```

Comment:
This is NEVER required. The DB auto inits itself - if you look at the code!!! If it hasn't been init'ed it will auto init when the first call is made to it!